postgrest/_async/request_builder.py

```diff
@@ -97,6 +167,34 @@ def __init__(
             self, session, path, http_method, headers, params, json
         )
 
+    def single(self):
```

Comment:
```suggestion
    def single(self) -> AsyncSingleRequestBuilder:
```missing return annotation